tests/integration/targets/mso_schema_site_anp_epg/tasks/main.yml

```diff
@@ -494,63 +457,6 @@
     - nm_query_non_anp.msg == "Provided anp 'ANP_3' does not exist at site level."
   when: version.current.version is version('4.0', '<') # no change in NDO4.0 because site will already be present when template is defined
 
-# USE A NON-EXISTING STATE
-- name: Non-existing state (normal_mode)
-  cisco.mso.mso_schema_site_anp_epg:
-    <<: *mso_info
-    schema: '{{ mso_schema | default("ansible_test") }}'
-    site: 'aws_{{ mso_site | default("ansible_test") }}'
-    template: Template 1
-    anp: ANP
-    epg: ansible_test_1
-    state: non-existing-state
-  ignore_errors: true
-  register: nm_non_existing_state
-
-- name: Verify non_existing_state
-  ansible.builtin.assert:
-    that:
-    - nm_non_existing_state is not changed
-    - nm_non_existing_state.msg == "value of state must be one of{{':'}} absent, present, query, got{{':'}} non-existing-state"
-
-# # USE A NON-EXISTING TEMPLATE
-- name: Non-existing template (normal_mode)
-  cisco.mso.mso_schema_site_anp_epg:
-    <<: *mso_info
-    schema: '{{ mso_schema | default("ansible_test") }}'
-    site: 'aws_{{ mso_site | default("ansible_test") }}'
-    template: non-existing-template
-    anp: ANP
-    epg: ansible_test_1
-    state: query
-  ignore_errors: true
-  register: nm_non_existing_template
-
-- name: Verify non_existing_template
-  ansible.builtin.assert:
-    that:
-    - nm_non_existing_template is not changed
-    - nm_non_existing_template.msg == "Provided template 'non-existing-template' does not exist. Existing templates{{':'}} Template1, Template2, Template3"
-
-# USE A NON-EXISTING SCHEMA
-- name: Non-existing schema (normal_mode)
-  cisco.mso.mso_schema_site_anp_epg:
-    <<: *mso_info
-    schema: non-existing-schema
-    site: 'aws_{{ mso_site | default("ansible_test") }}'
-    template: Template 1
-    anp: ANP
-    epg: ansible_test_1
-    state: query
-  ignore_errors: true
-  register: nm_non_existing_schema
-
-- name: Verify non_existing_schema
-  ansible.builtin.assert:
-    that:
-    - nm_non_existing_schema is not changed
-    - nm_non_existing_schema.msg == "Provided schema 'non-existing-schema' does not exist."
-
```

Comment:
Should we move these tests to physical site or are they done on physical sites somewhere else?

Reply:
You resolved this without a comment. Please explain why it shoudn't be moved to Physical site.

Reply:
I moved the code to `Physical site` and the `USE A NON-EXISTING STATE` task is not useful, so removed it from the test file.